lib/broccoli/ember-app.js

```diff
@@ -195,7 +195,7 @@ class EmberApp {
     let templatesTree = existsSync(templatesPath) ? new UnwatchedDir(templatesPath) : null;
 
     // do not watch bower's default directory by default
-    let bowerTree = this.project._watchmanInfo.enabled ? this.bowerDirectory : new UnwatchedDir(this.bowerDirectory);
+    let bowerTree = this.project._watchmanInfo.enabled ? this.bowerDirectory : new UnwatchedDir(this._resolveLocal(this.bowerDirectory));
```

Comment:
This will only resolve when watchman is not present/enabled.  We need to resolve it either way.

I think we want something like this will work:

```js
let bowerDirectory = this._resolveLocal(this.bowerDirectory);
let bowerTree = this.project._watchmanInfo.enabled ? bowerDirectory : new UnwatchedDir(bowerDirectory);
```